core/CronArchive/QueueConsumer.php

```diff
@@ -550,6 +555,23 @@ private function findSegmentForArchive(&$archive)
         return $this->segmentArchiving->isAutoArchivingEnabledFor($storedSegment);
     }
 
+
+    /**
+     * check if segments that contain dimensions that don't exist anymore
+     * @param $invalidatedArchive
+     * @return bool
+     */
+    private function isSegmentAvaible($invalidatedArchive)
+    {
+        try {
+            new Segment($invalidatedArchive['segment'], [$invalidatedArchive['idsite']]);
+           } catch (\Exception $e) {
+            $this->logger->debug("Segment is not valid anymore.");
```

Comment:
You could add some details to the log message, like the segment and idsite, so it's possible to check which segment it it caused.